### DISCUSSION TITLE
滚动条优化

### DISCUSSION THREAD

**Discussion by llwy142086** [WatsonYao/nian_app#101] [Q&A] [answered]
我的进展比较多，有几千条，但是用旁边的滚动条尝试快速滑动时发现超级难用，我的手指总是按不到那个灰色滑动条上（实在太窄了T^T），希望可以做得宽一点。
还有还有，我会给这些进展添加标签，但是标签筛选下没有滚动条和改变阅读顺序的选项，所以我很难看到时间比较久的进展（某个标签下我有一千多条T^T），可以在筛选界面也增加滚动条吗？

**Comment by WatsonYao** [comment, marked as answer]
1. 那个滑块目前无法拓宽，建议记本进展拆分一下，一个记本不适合放太多进展。推荐按时间段建立记本（比如 日常2024，阅读2024S01）这样查阅起来会比较方便。
2. 搜索那块，后续会添加一个时间倒序的按钮。
